dockstore-webservice/src/main/resources/migrations.1.13.0.xml

```diff
@@ -36,4 +36,33 @@
             <where>name = ''</where>
         </update>
     </changeSet>
+    <changeSet author="ajandu" id="replace_empty_text">
+        <!-- Replace any empty strings with NULL, same as above but for all columns except dbcreatedate, dbupdatedate, and id -->
+        <sql dbms="postgresql">
+            UPDATE user_profile SET company=NULL WHERE company='';
+            UPDATE user_profile SET email=NULL WHERE email='';
+            UPDATE user_profile SET location=NULL WHERE location='';
+            UPDATE user_profile SET name=NULL WHERE name='';
+            UPDATE user_profile SET token_type=NULL WHERE token_type='';
+            UPDATE user_profile SET username=NULL WHERE username='';
+            UPDATE user_profile SET onlineprofileid=NULL WHERE onlineprofileid='';
+            UPDATE user_profile SET bio=NULL WHERE bio='';
+            UPDATE user_profile SET avatarurl=NULL WHERE avatarurl='';
+            UPDATE user_profile SET link=NULL WHERE link='';
+        </sql>
+    </changeSet>
```

Comment:
very unlikely, but technically each transaction is a changeset so these two changes should be in the same changeset to avoid being ninjaed with an empty string between transactions